include/ews/ews.hpp

```diff
@@ -16676,6 +16679,26 @@ class contact final : public item
     // manager_mailbox
     // direct_reports
 
+    std::string get_user_smime_certificate() const
+    {
+	const auto node = xml().get_node("UserSMIMECertificate");
+	if (!node) {
+		return "";
+	}
+	auto base64binary = node->first_node("Base64Binary");
```

Comment:
You have to consider the namespace to access the need. Either by
```
auto base64binary = node->first_node_ns(internal::uri<>::microsoft::types(), "Base64Binary");
```
or
```
auto base64binary = node->first_node("t:Base64Binary");
```
Otherwise you don't get the node and the method will return the empty string.